.github/actions/fetch_ctk/action.yml

```diff
@@ -23,7 +23,7 @@ runs:
   using: composite
   steps:
     - name: Set up CTK cache variable
-      shell: bash --noprofile --norc -xeuo pipefail {0}
+      shell: bash -xeuo pipefail {0}
```

Comment:
For some reason the Git Bash that I installed really hates `--noprofile --norc` -- It becomes a no-op entirely (nothing happens when a job step launches it). However, this is not the case on GH-hosted Windows (build) runners. I have no clue why it is the case, but if the CI is all green I don't plan to dig further.

Reply:
It turns out that this weird ness is a symptom that I used the wrong bash executable. It should be `bin\bash.exe` (which runs in the same terminal) instead of the root `git-bash.exe` (which opens a new window when tested locally). Commit [a73564c](https://github.com/NVIDIA/cuda-python/pull/600/commits/a73564c5b423033a985e42ab18a44d96a54d3fea) shows that by fixing this we not only recover all `fetch_ctk` verbose output (critical for debugging!) but also can revert this hack. Now the custom Git Bash installation behaves like the one installed in the GH-hosted Windows images:
https://github.com/actions/runner-images/blob/5676b8b82520e84040b0e57569be0c0db7d92c4c/images/windows/scripts/build/Configure-Shell.ps1#L16

Reply:
> critical for debugging!

oh and this also fixes the silent error suppression -- if `fetch_ctk` fails, now it propagates correctly (PowerShell can check this via `$?`
https://github.com/NVIDIA/cuda-python/actions/runs/14808554816/job/41580483605